sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -931,6 +963,25 @@ public virtual async Task<Response<ConfigurationSettingsSnapshot>> CreateSnapsho
             }
         }
 
+        internal async Task<Response> CreateSnapshotAsync(string name, RequestContent content, ContentType contentType, RequestContext context = null)
```

Comment:
are these manually generated?

Looking at the autorest.md settings we have set public-clients: true however we are also creating a public client manually which seems like a conflict. Please remove that flag.

Reply:
These changes are still in progress, but I will look into why the autorest.md file is written that way. I asked around trying to learn how to internalize the dozens of unnecessary protocol methods being generated and this is what was shared with me as the solution.

Reply:
Got it. I will be happy to help you if you have questions. @m-nash may be able to assist us more here and appconfig may be in an interim state due to the intern project work. Here is where I would ultimately want AppConfig to be

1. Use the same autorest settings as ingestion (so use DPG generator w/o models and retrofit the convenience layer with it with or w/o making the protocol methods public (depending on what guidance architects give) and whatever is most efficient in terms of cost. 
2. Use HLC generator and be consistent with rest of older data plane SDKs that our team owns. I would prefer the top one because this is closest to what we tell our partners and we are so close to getting AppConfig there.

Reply:
@m-redding given we don't intend to expose protocol methods - we will have to move us back to the older flag generation1-convenience-client: true (this will move us back to HLC generator), however this should be fixed in a couple of months once we move to single generator. @m-nash please keep me honest here.

Reply:
Ok sounds good I'll fix that!